Resources/Prototypes/Species/moth.yml

```
@@ -58,17 +58,17 @@
       points: 1
       required: false
     HeadTop:
-      points: 1
+      points: 6
       required: true
       defaultMarkings: [ MothAntennasDefault ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)